src/Flysystem/StreamCommand/StreamOpenCommand.php

```diff
@@ -68,7 +68,7 @@ public static function run(
 
             $current->alwaysAppend = 'a' === $mode[0];
             if (is_resource($current->handle) && !$current->alwaysAppend) {
-                rewind($current->handle);
+                @rewind($current->handle);
```

Comment:
There might be a better way to handle this, but here a warning is dispatched for me:

```
Warning: rewind(): Stream does not support seeking  
```

Maybe a check would be better here?